src/PIF.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class PIF {
+    private String codes;
+    private String filename;
+
+    public PIF(String filename) {
+        this.filename = filename;
+        codes = "";
+        readFromFile();
+        System.out.println(codes);
+    }
+
+    private void readFromFile() {
```

Comment:
Same as before, maybe have a FileService, which there is a method that can read from a specific file and return a PIF class?